student.rb

```diff
@@ -1,18 +1,17 @@
+require_relative 'school_people'
+
 class Student
+  include School_Person
   attr_reader :age, :phase
   attr_accessor :name
 
   def initialize(options = {})
-    @phase = 1
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-  end
-
-  def offer_high_five
-    "High five!"
+    @phase = 1 #work into module 
+    @age = options.fetch(:age, 0) #add to module
+    @name = options.fetch(:name, "") #add to module
```

Comment:
Modules can't do initialize.  And your high five module probably shouldn't include initializing state anyway.  What else could you do that would allow you to pull these out and have them both here and in Educator?